tools/transformer/src/atlasTransformations.js

```diff
@@ -40,5 +40,21 @@ module.exports = function runTransformations(openapi) {
     ignoredModelNames
   );
 
+  // Temp workaround for
+  // https://jira.mongodb.org/browse/CLOUDP-166120
+  openapi.components.responses.noBody = {
+    content: {
+      "application/vnd.atlas.2023-01-01+json": {
+        example: "",
+      },
+    },
+    description: "This endpoint does not return a response body",
+  };
+
+  // Temp workaround for CLOUDP-168427
+  if (openapi.components.schemas.Error) {
+    openapi.components.schemas.Error.properties.parameters.items = {};
+  }
```

Comment:
why haven't we fixed this upstream?

Reply:
We did but we are waiting for fix to arrive in 3 weeks.

Reply:
do you have a ticket on the fix upstream?